src/components/ForgotPassword/ForgotPassword.css

```diff
@@ -0,0 +1,6 @@
+.a6y-react-auth__forgot-password {
```

Comment:
Do the other forms have the same (or quite similar) style? If so, it could be better to have a one class that would be shared by all forms. Moreover, I believe that there should not be any styling differences between `SignIn`, `SignUp` or `ForgotPassword` forms and forgot password doesn't have anything unique. So I think that `.a6y-react-auth__forgot-password` is not needed to be styled. It's OK that the class is added to the div, but it doesn't have any styling there.

I mean, that if we use some class named after some component then I would say that class is very specific and the component has a unique styling.